prover/crypto/keccak/subtle/xor_arm64.s

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 The Go Authors. All rights reserved.
```

Comment:
Likewise, we should remove the header or avoid these optimization if we copy non-apache code.